apps/next/components/Subdomains/SubdomainContent.tsx

```diff
@@ -83,7 +83,9 @@ const AddSubdomain = ({
     async onSuccess(data) {
       await data.wait(1);
       refetchFn();
-      // console.log('Success setSubnodeRecord', data)
+      console.log('Success setSubnodeRecord', data)
```

Comment:
Worth commenting out or removing console logs for every commit so we don't pollute the console like crazy for every change

Reply:
I agree. Resolved!